source/Octopus.Client.Tests/Operations/RegisterMachineOperationFixture.cs

```diff
@@ -153,7 +152,8 @@ public async Task ShouldUpdateExistingMachineWhenForceIsEnabled()
             await client.Received().Update("/machines/whatever/1", Arg.Is<MachineResource>(m =>
                 m.Id == "machines/84"
                 && m.Name == "Mymachine"
-                && m.EnvironmentIds.First() == "environments-2")).ConfigureAwait(false);
+                && m.EnvironmentIds.First() == "environments-2"),
+                Arg.Any<object>(), Arg.Any<CancellationToken>()).ConfigureAwait(false);
```

Comment:
Both of these arguments are now expected as part of the newly added `ICreate.Create(TResource, object, CancellationToken)`